libraries/AP_DroneCAN/AP_DroneCAN.cpp

```diff
@@ -391,6 +393,9 @@ void AP_DroneCAN::init(uint8_t driver_index, bool enable_filters)
 #if HAL_MOUNT_XACTI_ENABLED
     AP_Mount_Xacti::subscribe_msgs(this);
 #endif
+#if AP_TEMPERATURE_SENSOR_DRONECAN_ENABLED
+    AP_TemperatureSensor_DroneCAN::subscribe_msgs(this);
```

Comment:
why not do this in AP_TemperatureSensor_DroneCAN

Reply:
I'm not sure how to do it the other way and get the driver object to I can add a subscriber.